CHANGELOG.md

```diff
@@ -15,6 +15,8 @@
 #### :rocket: New Feature
 
 - Inlay Hints (experimetal). `rescript.settings.inlayHints.enable: true`
+- Code Lenses for functions (experimetal). `rescript.settings.codeLens: true`
```

Comment:
Should is say somewhere that it's off by default?

Reply:
Done.